proto/storage/notification.proto

```diff
@@ -12,8 +12,8 @@ package storage;
 //
 // Refer to v1.Notification for a more detailed doc.
 message Notification {
```

Comment:
Naming is hard.

I'm worried that `Notification` might be a bit too broad (not just the proto name, also how you refer to the concept in the service). We also notifiers, which are today meant to _notify_ about policy violations and alerts, so I'd rather make the distinction more "in your face". 

A few ideas:
* ProductNotification, similar to recently introduced product usage. Can even live under `/product`!
* CentralNotification
* CentralEvent

Reply:
I think I'd prefer `CentralNotification`. I think we should keep "notification" rather than "event" because it will also be called "notification" in the UI.

Reply:
> Does CentralEvent describe the thing more clearly than CentralNotification does?

I'm not sure that it does. When the notifications are generated from log events maybe, but the API is open to other types in the future. Also imo fields like `hint` fit better with notification than event.

> From viewpoint of customer problem solving, what do you think is the proportion of times a notification announces a problem versus times a problem becomes apparent for other reasons, and then a customer filters to find relevant events?

That's a good question, I don't know. AFAIK both are valid scenarios.

> To put it another way, declarative configuration health generalized the precedent of integrations health. It is hard for me to escape the idea that a practical way to meet the current need might take another step forward the precedent of health errors and diagnostic bundle. Even if we confirm the decision to break away from previous precedents, it does not require notifications to be the concept in storage and API.

Sorry what precedents do you mean?

Reply:
I'm currently favoring `ProductNotification` because it gives us a natural way to have the API namespaced under `/v1/product/notifications`. Central code would live under `central/product/notifications`. I'll sleep on it and do the refactoring then.

Reply:
Precedents are if I am guessing correctly that information in integration health and declarative configuration health comes from logs?

Reply:
For posterity, the naming has been decided to be `/v1/administration/events`, with the proto itself being called `AdministrationEvent`.